### ISSUE TITLE
Invalid launch.json entries + unable to debug

### ISSUE THREAD

**Issue by ekkis** [Microsoft/vscode-chrome-debug#649]
I've followed the instructions at:

https://medium.com/@auchenberg/live-edit-and-debug-your-react-apps-directly-from-vs-code-without-leaving-the-editor-3da489ed905f

which had me install the extension and use the `launch.json` configuration shown below:

 ```
{
            "name": "Chrome",
            "type": "chrome",
            "request": "launch",
            "url": "http://localhost:3000",
            "webRoot": "${workspaceRoot}/src"
 }
```
I have version 4.3.0 of the "Debugger for Chrome" extension installed but when I enter the given configuration I get the following errors:

“Property url is not allowed”
“Property webRoot is not allowed”

and more importantly: 

“The debug type is not recognized. Make sure that you have a corresponding debug extension installed and that it is enabled.”

my aim is to debug this example code from TradingView, which is a React app (I don't know if it was created with create-react-app):

https://github.com/tradingview/charting-library-examples

but after launching the app with `npm start`, I click F5 on VS Code and get the error:

> Attribute 'program' does not exist ('/Users/ekkis/dev/NDX/charting-library-examples/react-example/start').

can anyone help?

**Comment by roblourens** [comment]
Did you reload after installing the Chrome extension? The JSON validation errors make it sound like it isn't installed.

**Comment by ekkis** [comment]
yes but just for kicks I restarted again.  it still complains in the same fashion

**Comment by ekkis** [comment]
damn.  you're right, it wasn't installed.  it appeared in my list of extensions but when I clicked on it, the Install button was green and needed to be pressed.  I had originally clicked on the link the article, which I guess downloaded but didn't install the extension.  thank you!
